andela/ah-code-titans-frontend
# #161967083 update TravisCI to run tests.

#### What does this PR do?

- Setup TravisCI for automatic builds and running of tests.

#### Description of tasks to be completed?

- Add project repository to [travis-ci.org](https://travis-ci.org/andela/ah-code-titans-frontend)

- Add `.travis.yml` file to the repository to hold configuration settings for builds on Travis.

#### How should this be manually tested?

- Open the build for this branch on Travis [here](https://travis-ci.org/andela/ah-code-titans-frontend/).
- Notice that the build is already passing.
- Restart the build.
- Build should pass.

#### Screenshot(s)
<img width="930" alt="screenshot 2019-01-08 at 16 12 50" src="https://user-images.githubusercontent.com/25426934/50833149-66dc9b80-1361-11e9-921a-38b59928cdd3.png">

#### What are the relevant pivotal tracker stories?
[#161967083](https://www.pivotaltracker.com/story/show/161967083)

#### Checklist
